acme.sh

```diff
@@ -0,0 +1,94 @@
+# Common acme functions
+
+fn abortonerr(status, msg) {
+	if $status != "0" {
+		io_println($msg)
+		exit(1)
+	}
+}
+
+fn acme_read(rpath) {
+	var fpath = "acme/"+$winid+"/"+$rpath
+
+	var content, status <= 9p read $fpath
```

Comment:
this line spaced code is irritating me. I'll change the nashfmt to not put this spaces anymore,